Paper.tex

```diff
@@ -168,12 +166,18 @@ \subsection{Which History?}
 
 This would mean that past a given point in time (block), multiple states of the system may coexist: some nodes believing one block to contain the canonical transactions, other nodes believing some other block to be canonical, potentially containing radically different or incompatible transactions. This is to be avoided at all costs as the uncertainty that would ensue would likely kill all confidence in the entire system.
 
-The scheme we use in order to generate consensus is a simplified version of the GHOST protocol introduced by \cite{cryptoeprint:2013:881}. This process is described in detail in section \ref{ch:ghost}.
+Since the \textit{Paris} hard fork, reaching consensus on new blocks is managed by a protocol called the \textit{Beacon Chain}.
+It is known as the \textit{consensus layer} of Ethereum, and it defines the rules for determining the canonical history of Ethereum blocks.
+This document describes the \textit{execution layer} of Ethereum.
+The execution layer defines the rules for interacting with and updating the state of the Ethereum virtual machine.
+The consensus layer is described in greater detail in the \href{https://github.com/ethereum/consensus-specs}{consensus specifications}.
+How the consensus layer is used to determine the canonical state of Ethereum is discussed in section \ref{ch:blocktree_to_blockchain}.
 
 Sometimes, a path follows a new protocol from a particular height (block number).
```

Comment:
Starting from Shanghai, hard forks are triggered by block time rather than block height.

Reply:
I'll add some detail on how forks can be triggered

Reply:
added!